config/settings.yml

```diff
@@ -0,0 +1,46 @@
+---
+:ems:
+  :ems_amazon:
+    :disabled_regions: []
+    # disable regions by their keys found in app/models/manageiq/providers/amazon/regions.rb e.g.
+    # uncomment the following to disable gov-west-1 region
```

Comment:
You are right about the `uncomment` statement. 

> Note that this comment will not appear in Advanced Settings

Yeah, I know, but I havent found a better place to add some configuration hints. Same as [here](https://github.com/ManageIQ/manageiq/blob/master/config/settings.yml#L121)

I think it's better than nothing. And advanced users _might_ find their way by googleing or looking at the source. And if its only support staff of ours.

Would it be better to approach the docs team to add that to the docs?